src/components/BlogComponent/Blogs.tsx

```diff
@@ -0,0 +1,66 @@
+import { Card, CustomFlowbiteTheme } from "flowbite-react";
+import AsyncawaitBlogImg from "../../assets/BlogImages/asyncawaitImg.jpg";
+import IPaasBlogImg from "../../assets/BlogImages/IPaasImg.jpg";
+import HibernateBlogImg from "../../assets/BlogImages/hqlImg.jpg";
+
+const Blogs = () => {
+  const BlogData = [
+    [
+      "07/04",
+      AsyncawaitBlogImg,
+      "Benefits Of Async/Await",
+      "Asynchronous functions are a good and bad thing in JavaScript. The good side is that asynchronous functions are non-blocking and",
+    ],
+    [
+      "07/17",
+      IPaasBlogImg,
+      "Key Considerations Of IPaaS",
+      "Digital transformation requires cloud appropriate adoption, legacy IT systems modernization, and Agile-based methodologies for faster",
+    ],
+    [
+      "07/22",
+      HibernateBlogImg,
+      "Hibernate Query Language",
+      "In this tutorial, we will discuss the Hibernate Query Language. HQL is an object-oriented query language. Hibernate Query",
+    ],
+  ];
+
```

Comment:
Instead of using Array of Arrays use Array of Objects and give proper keys names in object. For example - [{name:'ABC', date:'11-12-2014'},{name:'ABC', date:'11-12-2014'}].